tests/behat/features/capabilities/groups/flexible/groups-flexible-sidebar-blocks.feature

```diff
@@ -18,4 +18,3 @@ Feature: Flexible groups sidebar blocks are correctly displayed
 
     Then I should see "Upcoming events" in the "Sidebar second"
     And I should see "Newest topics" in the "Sidebar second"
-    And I should see "Newest Members" in the "Sidebar second"
```

Comment:
@rochek03 since we explicitly change this logic, I think we can also replace this row with `I should not see "Newest Members"`